Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM centos:7
+ENV container docker
+
+# Set locale manually on build (docker defaults to POSIX which causes initdb to set the wrong db encoding, MIQ must have UTF8)
+# Once systemd is online, it will set locale based /etc/locale.conf
+
+# Only needed if building with docker-1.8
+ENV LANG en_US.UTF-8  
+
+# Install EPEL repo, yum necessary packages for the build without docs, clean all caches (no DNF)
+RUN rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && yum -y install --setopt=tsflags=nodocs nodejs tar sudo git memcached postgresql-devel postgresql-server libxml2-devel libxslt-devel patch gcc-c++ openssl-devel gnupg curl which net-tools libyaml-devel autoconf readline-devel libffi-devel bzip2 automake libtool bison sqlite-devel ; yum clean all
```

Comment:
It sounds like you're missing some environment variables... sqlite should not be required in a build since it is a test dependency. https://github.com/ManageIQ/manageiq/blob/master/Gemfile#L103-L107